dotcom-rendering/src/web/components/CpscottComponent.tsx

```diff
@@ -0,0 +1,64 @@
+import { css } from '@emotion/react';
+import { headline, neutral } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import { Hide } from './Hide';
+import { QuoteIcon } from './QuoteIcon';
+
+const scottAvatarStyles = css`
+	width: 72px;
+	height: 72px;
+	border-radius: 100%;
+	overflow: hidden;
+	background-color: #fcebde;
+`;
+
+const scottPortraitStyles = css`
+	width: 82px;
+`;
+
+const scottTextStyles = css`
+	${headline.xxxsmall()}
+	font-size: 14px;
+	line-height: 16px;
+`;
+
+const quoteLineStyles = css`
+	color: ${neutral[46]};
+`;
+
+export const CPScottComponent = () => {
```

Comment:
non-blocking:

Probably worth keeping the file and the component the same name. 

[We can lint this](https://github.com/github/eslint-plugin-custom-elements/blob/main/docs/rules/file-name-matches-element.md) (maybe another PR).

[Vue has something similar too](https://eslint.vuejs.org/rules/match-component-file-name.html).

A small question on naming - can you find the pattern as to  when we use the `Component` prefix? Maybe just random.